tests/cases/unittests/tsconfigParsing.ts

```diff
@@ -120,5 +148,15 @@ namespace ts {
                     ]
                 }`);
         });
+
+        it("ignore dotted files and folders", () => {
+            assertParseFileList(
+                `{}`,
+                "tsconfig.json",
+                "/apath",
+                ["/apath/test.ts", "/apath/.git/a.ts", "/apath/.b.ts", "/apath/..c.ts"],
+                ["/apath/test.ts"]
+            )
+        })
```

Comment:
Shouldn't there be a test "allow dotted files and folders when explicitly requested" ?